### ISSUE TITLE
Error in tolower(validBSgenome(getGenome(ArchRProj))@provider_version) :    no slot of name "provider_version" for this object of class "BSgenome"

### ISSUE THREAD

**Issue by sylestiel** [GreenleafLab/ArchR#393]
proj4 <- addArchRAnnotations(ArchRProj = proj4, collection = "EncodeTFBS")
ArchR logging to : ArchRLogs/ArchR-addArchRAnnotations-1ccc29e3bc5e-Date-2020-11-03_Time-15-31-58.log
If there is an issue, please report to github with logFile!
Error in tolower(validBSgenome(getGenome(ArchRProj))@provider_version) : 
  no slot of name "provider_version" for this object of class "BSgenome"

I do not understand the problem let alone how to address it.

Any suggestions? I tried downloading 
wget https://jeffgranja.s3.amazonaws.com/ArchR/Annotations/LOLA-Mm10-v1.Anno
How do I get the R program to recognize it?

Thank you!

**Comment by mzhibo** [comment]
had the same issue. Did addArchRGenome miss something?

**Comment by MRogne007** [comment]
Hi, 
I am experiencing the same error listed in issue #393  when I am using the AddArchRAnnotation function.

We have updated the BSgenome version. For me createGenomeAnnotation (as discussed in #416) works fine. 

I also attach the error message I obtain upon attempting to run the function: 

<img width="800" alt="Screenshot 2020-12-22 at 10 48 56" src="https://user-images.githubusercontent.com/68116041/102874860-5127ea00-4443-11eb-84cc-8b26de6cda65.png">


```
> sessionInfo()
R version 4.0.3 (2020-10-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.5

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
 [1] grid      parallel  stats4    stats     graphics  grDevices utils     datasets  methods  
[10] base     

other attached packages:
 [1] BSgenome.Hsapiens.UCSC.hg38_1.4.3        LOLA_1.20.0                             
 [3] chromVAR_1.12.0                          magick_2.5.2                            
 [5] chromVARmotifs_0.2.0                     motifmatchr_1.12.0                      
 [7] presto_1.0.0                             ggrepel_0.8.2                           
 [9] circlize_0.4.11                          ComplexHeatmap_2.6.2                    
[11] umap_0.2.7.0                             gtable_0.3.0                            
[13] scran_1.18.2                             SingleCellExperiment_1.12.0             
[15] pheatmap_1.0.12                          harmony_1.0                             
[17] Rcpp_1.0.5                               uwot_0.1.10                             
[19] ggridges_0.5.2                           gridExtra_2.3                           
[21] nabor_0.5.0                              Seurat_3.9.9.9010                       
[23] TxDb.Hsapiens.UCSC.hg38.knownGene_3.10.0 GenomicFeatures_1.42.1                  
[25] org.Hs.eg.db_3.12.0                      AnnotationDbi_1.52.0                    
[27] BSgenome_1.58.0                          rtracklayer_1.50.0                      
[29] Biostrings_2.58.0                        XVector_0.30.0                          
[31] HDF5Array_1.18.0                         DelayedArray_0.16.0                     
[33] hdf5r_1.3.3                              progress_1.2.2                          
[35] ArchR_1.0.1                              magrittr_2.0.1                          
[37] rhdf5_2.34.0                             Matrix_1.2-18                           
[39] data.table_1.13.4                        SummarizedExperiment_1.20.0             
[41] Biobase_2.50.0                           GenomicRanges_1.42.0                    
[43] GenomeInfoDb_1.26.2                      IRanges_2.24.1                          
[45] S4Vectors_0.28.1                         BiocGenerics_0.36.0                     
[47] MatrixGenerics_1.2.0                     matrixStats_0.57.0                      
[49] ggplot2_3.3.2                           

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.1              R.methodsS3_1.8.1           tidyr_1.1.2                
  [4] bit64_4.0.5                 knitr_1.30                  R.utils_2.10.1             
  [7] irlba_2.3.3                 rpart_4.1-15                KEGGREST_1.30.1            
 [10] TFBSTools_1.28.0            RCurl_1.98-1.2              generics_0.1.0             
 [13] callr_3.5.1                 cowplot_1.1.0               usethis_2.0.0              
 [16] RSQLite_2.2.1               RANN_2.6.1                  future_1.21.0              
 [19] bit_4.0.4                   spatstat.data_1.5-2         xml2_1.3.2                 
 [22] httpuv_1.5.4                assertthat_0.2.1            DirichletMultinomial_1.32.0
 [25] xfun_0.19                   hms_0.5.3                   evaluate_0.14              
 [28] promises_1.1.1              fansi_0.4.1                 caTools_1.18.0             
 [31] dbplyr_2.0.0                igraph_1.2.6                DBI_1.1.0                  
 [34] htmlwidgets_1.5.3           purrr_0.3.4                 ellipsis_0.3.1             
 [37] RSpectra_0.16-0             dplyr_1.0.2                 annotate_1.68.0            
 [40] biomaRt_2.46.0              deldir_0.2-3                sparseMatrixStats_1.2.0    
 [43] vctrs_0.3.6                 remotes_2.2.0               Cairo_1.5-12.2             
 [46] ROCR_1.0-11                 abind_1.4-5                 withr_2.3.0                
 [49] sctransform_0.3.1           GenomicAlignments_1.26.0    prettyunits_1.1.1          
 [52] goftest_1.2-2               cluster_2.1.0               seqLogo_1.56.0             
 [55] lazyeval_0.2.2              crayon_1.3.4                edgeR_3.32.0               
 [58] pkgconfig_2.0.3             labeling_0.4.2              nlme_3.1-151               
 [61] pkgload_1.1.0               devtools_2.3.2              rlang_0.4.9                
 [64] globals_0.14.0              lifecycle_0.2.0             miniUI_0.1.1.1             
 [67] BiocFileCache_1.14.0        rsvd_1.0.3                  rprojroot_2.0.2            
 [70] polyclip_1.10-0             lmtest_0.9-38               Rhdf5lib_1.12.0            
 [73] zoo_1.8-8                   GlobalOptions_0.1.2         processx_3.4.5             
 [76] png_0.1-7                   viridisLite_0.3.0           rjson_0.2.20               
 [79] bitops_1.0-6                R.oo_1.24.0                 KernSmooth_2.23-18         
 [82] rhdf5filters_1.2.0          blob_1.2.1                  DelayedMatrixStats_1.12.1  
 [85] shape_1.4.5                 stringr_1.4.0               parallelly_1.21.0          
 [88] readr_1.4.0                 CNEr_1.26.0                 beachmat_2.6.2             
 [91] scales_1.1.1                hexbin_1.28.1               memoise_1.1.0              
 [94] plyr_1.8.6                  ica_1.0-2                   zlibbioc_1.36.0            
 [97] compiler_4.0.3              dqrng_0.2.1                 tinytex_0.27               
[100] RColorBrewer_1.1-2          clue_0.3-58                 fitdistrplus_1.1-3         
[103] Rsamtools_2.6.0             cli_2.2.0                   listenv_0.8.0              
[106] patchwork_1.1.0             pbapply_1.4-3               ps_1.5.0                   
[109] MASS_7.3-53                 mgcv_1.8-33                 tidyselect_1.1.0           
[112] stringi_1.5.3               yaml_2.2.1                  BiocSingular_1.6.0         
[115] askpass_1.1                 locfit_1.5-9.4              tools_4.0.3                
[118] future.apply_1.6.0          rstudioapi_0.13             TFMPvalue_0.0.8            
[121] bluster_1.0.0               farver_2.0.3                Rtsne_0.15                 
[124] digest_0.6.27               BiocManager_1.30.10         pracma_2.2.9               
[127] shiny_1.5.0                 scuttle_1.0.3               later_1.1.0.1              
[130] RcppAnnoy_0.0.18            httr_1.4.2                  rhandsontable_0.3.7        
[133] colorspace_2.0-0            XML_3.99-0.5                fs_1.5.0                   
[136] tensor_1.5                  reticulate_1.18             splines_4.0.3              
[139] statmod_1.4.35              spatstat.utils_1.17-0       shinythemes_1.1.2          
[142] plotly_4.9.2.1              sessioninfo_1.1.1           xtable_1.8-4               
[145] poweRlaw_0.70.6             jsonlite_1.7.2              spatstat_1.64-1            
[148] testthat_3.0.1              R6_2.5.0                    pillar_1.4.7               
[151] htmltools_0.5.0             mime_0.9                    DT_0.16                    
[154] glue_1.4.2                  fastmap_1.0.1               BiocParallel_1.24.1        
[157] BiocNeighbors_1.8.2         codetools_0.2-18            pkgbuild_1.2.0             
[160] lattice_0.20-41             tibble_3.0.4                curl_4.3                   
[163] leiden_0.3.6                gtools_3.8.2                GO.db_3.12.1               
[166] openssl_1.4.3               survival_3.2-7              limma_3.46.0               
[169] rmarkdown_2.5               desc_1.2.0                  munsell_0.5.0              
[172] GetoptLong_1.0.5            GenomeInfoDbData_1.2.4      reshape2_1.4.4 
```

**Comment by rcorces** [comment]
Since this issue is closed, lets address this in #416 

I mentioned that this would be a problem in this post https://github.com/GreenleafLab/ArchR/issues/416#issuecomment-741964638

**Comment by rcorces** [comment]
@MRogne007 - see https://github.com/GreenleafLab/ArchR/issues/416#issuecomment-749581029 for the solution
